src/prefect/projects/steps/pull.py

```diff
@@ -17,7 +27,21 @@ def git_clone_project(
     repository: str, branch: Optional[str] = None, access_token: Optional[str] = None
 ) -> dict:
     """
-    Just a repo name will be assumed GitHub, otherwise provide a full repo_url.
+    Clones a git repository into the current working directory.
+    If only a repo name is provided GitHub will be assumed, otherwise provide a full repo_url for the
+    repository.
```

Comment:
I don't think we're doing this right now

Reply:
@desertaxle updated